### ISSUE TITLE
[HOLD for payment 2023-07-18] Codebase uses constant ACCESSIBILITY_ROLE_TEXT which does not exists

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#22764]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
Note: This is a code level bug related to accessibility role, so it does not affect user workflow.
1. Inspect the code here: https://github.com/Expensify/App/blob/38890e2e3c32b52dad11a7dfc09494924519946b/src/components/TextInput/TextInputLabel/index.js#L20

## Expected Result:
accessibilityRole is being set to `CONST.ACCESSIBILITY_ROLE_TEXT` which does not exists in the codebase

## Actual Result:
 It should be set to `CONST.ACCESSIBILITY_ROLE.TEXT` instead

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.39-3
**Reproducible in staging?:** n/a
**Reproducible in production?:** n/a
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation
**Expensify/Expensify Issue URL:**
**Issue reported by:** @honnamkuan 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1689072044290999

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01a18aafc7aae95a5a</li>
        <li>Upwork Job ID: 1679196736958214144</li>
        <li>Last Price Increase: 2023-07-12</li>
    </ul>
</details>

**Comment by situchan** [comment]
I had [commented](https://github.com/Expensify/App/pull/20363#discussion_r1259554307) in offending PR and follow-up [PR](https://github.com/Expensify/App/pull/22732) was raised already. This can be closed.

**Comment by mountiny** [comment]
Handled here https://github.com/Expensify/App/pull/22732

**Comment by mountiny** [comment]
Could we pay $250 to @honnamkuan for reporting this?

**Comment by NicMendonca** [comment]
@honnamkuan can you please apply here: https://www.upwork.com/jobs/~01a18aafc7aae95a5a

**Comment by NicMendonca** [comment]
offer sent!
